src/Livewire/PeriodSelector.php

```diff
@@ -4,27 +4,23 @@
 
 use Illuminate\Contracts\Support\Renderable;
 use Illuminate\Support\Facades\View;
-use Livewire\Attributes\Url;
 use Livewire\Component;
 
 /**
  * @internal
  */
 class PeriodSelector extends Component
 {
-    /**
-     * The selected period.
-     *
-     * @var '1_hour'|'6_hours'|'24_hours'|'7_days'
-     */
-    #[Url]
-    public string $period = '1_hour';
+    use Concerns\HasPeriod;
 
     /**
      * Render the component.
      */
     public function render(): Renderable
     {
-        return View::make('pulse::livewire.period-selector');
+        return View::make('pulse::livewire.period-selector', [
+            'periods' => $this->periods(),
+            'current' => $this->period,
```

Comment:
I preferred to send the current `$period` with `$current` to Blade so there is no name conflict [here](https://github.com/laravel/pulse/pull/411/files#diff-af1ecedb949e061764d9a9ffd84c20d4ae3f09498809ba68d838c26168a89b04R15)